src/constants.js

```diff
@@ -0,0 +1,196 @@
+export default {
```

Comment:
I think this is the same as JSON for the minifier. It wouldn't be able to remove dead definitions; I'm curious and I'd bet it also wouldn't fix the problem you were having. 

If it doesn't, what we want is 

```
export const EVENTS = {...}; 
export const TARGETING_KEYS = {...};
...
```

or even better would be 

```
export const EVENT_AUCTION_INIT = 'auctionInit';
...
export const TARGETING_KEY_BIDDER = 'hb_bidder';
...
```

but of course the latter is a lot more work. I'm sorry I didn't make this clearer earlier.